### PR TITLE
OSDOCS#9085: Custom domain docs for managed openshift console oauth and downloads

### PR DESCRIPTION
**Pull request by laubai**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-9085

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:
- Cloud Experts tutorial content; QE review is not required

### CODE REVIEW

File: cloud_experts_tutorials/cloud-experts-rosa-osd-change-default-domain.adoc [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+In our case, the hostname is `a234gsr3242rsfsfs-1342r624.us-east-1.elb.amazonaws.com`.
+
+Save this value for later, as we will need it to configure DNS records for our new component route hostnames.
+
+== Add component route DNS records to your hosting provider
+
+In your hosting provider, add DNS records that map the `CNAME` of your new component route hostnames to the load balancer hostname we found in the previous step.
+
+.Need an image for this
+image::[Picture goes here]
```
Comment (hectorakemp) [inline cloud_experts_tutorials/cloud-experts-rosa-osd-change-default-domain.adoc:170]: We can probably leave this part out to start with

Comment (laubai) [reply cloud_experts_tutorials/cloud-experts-rosa-osd-change-default-domain.adoc:170]: oops XD sorry about that

**Comment by bmcelvee** [comment]
LGTM!

**Pull request merged**